libp2p/identity/identity.go

```diff
@@ -0,0 +1,134 @@
+package identity
```

Comment:
1. I really like the location of this now.
2. My biggest question, as I'm reading this, are "what functions are available". Try running `make go_doc`!